python/cuspatial/cuspatial/__init__.py

```diff
@@ -28,5 +29,4 @@
 )
 from .io.geopandas import from_geopandas
 
-__version__ = get_versions()["version"]
-del get_versions
+__version__ = "23.06.00"
```

Comment:
Please verify that `ci/release/update-version.sh` is correct.
```suggestion
__version__ = "23.08.00"
```

Reply:
I'm thinking we might want to do those changes in another PR.

Typically we try to limit the scope of these PRs to just fix the conflicts.

Reply:
Unless this change is part of the conflict

Reply:
`update-version.sh` is correct, I just assumed git chose the correct versions in the merge commit.